solidity-frontend/src/main/java/org/sonarsource/solidity/frontend/Utils.java

```diff
@@ -49,13 +49,14 @@ public static boolean isCommentSignificant(String token) {
     return matcher.find();
   }
 
-  public static String returnFunctionSignature(FunctionDefinitionContext ctx) {
+  /*public static String returnFunctionSignature(FunctionDefinitionContext ctx) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=39&issues=AWSE3JtQsk71k7lwfTUK&open=AWSE3JtQsk71k7lwfTUK)

[ISSUE_KEY]: AWSE3JtQsk71k7lwfTUK